src/Skeleton/BaseController.php

```diff
@@ -274,7 +299,7 @@ protected function setStatusCode($statusCode)
      */
     protected function respondWithItem($item)
     {
-        $resource = new Item($item, $this->transformer);
+        $resource = new Item($item, $this->transformer, $this->resourceKey);
```

Comment:
There are some cases when you will have different keys on index, update, create methods.
Thats why its better to leave respondWithItem and respondWithCollection just with $this->resourceKey, and singular | plural - in create + update methods.

In my case - I can easy rewrite create method to make my key singular.

And, yeah, I think y're right - resourceKey should be singular by default.